buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -21,7 +21,7 @@ object Versions {
 
     const val compose = "1.0.0-rc02"
     const val composeNavigation = "2.4.0-alpha04"
-    const val composeConstraintLayout = "1.0.0-alpha07"
+    const val composeConstraintLayout = "1.0.0-alpha08"
```

Comment:
Huh, can't find release notes for this 🤔 [Release notes page](https://developer.android.com/jetpack/androidx/releases/constraintlayout#constraintlayout-compose-1.0.0-alpha07)

Reply:
I found it here https://maven.google.com/web/index.html?q=Constraint#androidx.constraintlayout:constraintlayout-compose:1.0.0-alpha08

Reply:
Found it https://github.com/androidx/constraintlayout/releases/tag/compose-1.0.0-alpha08